examples/elasticsearch-incremental/elasticsearch-incremental-blueprint.yaml

```diff
@@ -0,0 +1,542 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata: 
+  name: elasticsearch-incremental-blueprint 
+actions:
+  backup:
+    outputArtifacts:
+      s3Snap:
+        keyValue:
+          repoPath: '{{ .Phases.setupPhase.Output.repoPath }}'
+          snapshotName: '{{ .Phases.setupPhase.Output.snapshotName }}'
+          esClusterName: '{{ .Phases.setupPhase.Output.esClusterName }}'
+          esClusterNamespace: '{{ .Object.metadata.namespace }}'
+    phases:   
+    - func: KubeTask
+      name: setupPhase
+      objects:
+        elasticSecret:
+          kind: Secret
+          name: '{{ .Object.metadata.name }}-es-elastic-user'
+          namespace: '{{ .Object.metadata.namespace }}' 
+      args:        
+        image: ghcr.io/kanisterio/kanister-kubectl-1.18:0.81.0
+        command:
+        - bash
+        - -x
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          debug=0
+          function dbg
+          {
+              if [[ $debug -eq 1 ]]
+              then 
+                echo "$(date "+%Y-%m-%d %H:%M:%S.%3N") - $0 - $*" >> /tmp/k10_debug.log
+              fi
+          }
+          function bp_echo
+          {
+            # use 
+            # kubectl logs -n kasten-io -l component=kanister --tail=10000 -f | ggrep -o -P '(?<=Elasticbps3echobegin).*(?=Elasticbps3echoend)' 
+            # to grab the logs in kanisters 
+            echo "Elasticbps3echobegin backup-action: $* Elasticbps3echoend"
+          }            
```

Comment:
If you just echo then you can't grab easily the output of the blueprint in the kanister logs

Reply:
There was this field (`logKind`) introduced as part of [this PR](https://github.com/kanisterio/kanister/pull/1497) that can be used to figure out all the logs that were generated from kanister function pods (`DataPath`). Do you  think we can rely on that and remove this?
Because the phase is already too lengthy and the things that we are doing in this function can be done in place as well, where we are calling `bp_echo`. We don't follow this practice of having another log function in the BP.
WDYT?